src/renderer/tvgAnimation.cpp

```diff
@@ -20,33 +20,15 @@
  * SOFTWARE.
  */
 
-#include "tvgCommon.h"
 #include "tvgFrameModule.h"
-#include "tvgPaint.h"
 #include "tvgPicture.h"
+#include "tvgLottieLoader.h"
```

Comment:
this file doesn't refer: 
-- #include "tvgLottieLoader.h"

-- #include "tvgPicture.h"   //can be included in tvgAnimation.h

Reply:
Without `tvgLottieLoader.h`, the `LottieLoader` cannot be found

> identifier "LottieLoader" is undefinedC/C++(20)
```cpp
if (static_cast<LottieLoader*>(pImpl->picture->pImpl->loader)->override(slot)) {
```

Reply:
Please double-check it, I couldn't see any LottieLoader usage in tvgAnimation.cpp.
Even the call is belonged to tvgLottieAnimation.cpp, not tvgAnimation.cpp

Reply:
Oops, I checked wrong file. that's correct.